### ISSUE TITLE
Run code from eeprom

### ISSUE THREAD

**Issue by raspiduino** [billroy/bitlash#62]
Hello! I have tried bitlash and it can be more wonderful if your program can execute code from eeprom. Please add this feature to the program! Thank you so much!

**Comment by njh** [comment]
> execute code from eeprom

Do you mean an external EEPROM?
(functions are normally stored in the internal eeprom)

**Comment by raspiduino** [comment]
Nah, it has been 2 years since I asked that, and I already realized that it was a stupid idea.

Thanks anyway :))
